astropy/modeling/functional_models.py

```diff
@@ -440,6 +440,99 @@ def inverse(self):
         return inv
 
 
+class Sersic1D(Fittable1DModel):
+    """
+    One dimensional Sersic surface brightness profile.
+
+    Parameters
+    ----------
+    amplitude : float
+        Central surface brightness, within r_eff. 
+    r_eff : float
+        Effective (half-light) radius
+    n : float
+        Sersic Index.
+
+    
+    See Also
+    --------
+    Gaussian1D, Moffat1D, Lorentz1D
+
+    Notes
+    -----
+    Model formula:
+
+    .. math::
+
+        I(r)=I_e\\exp\\left[-b_n\left(\\frac{r}{r_{e}}\\right)^{(1/n)}-1\\right]
```

Comment:
If you use a "raw string", (i.e. put `r` before the `"""` above) you don't have to use the double slashes through here.  Might be a little more readable.